srcpkgs/cargo/template

```diff
@@ -10,8 +10,7 @@ maintainer="Enno Boland <gottox@voidlinux.eu>"
 license="MIT, Apache-2.0"
 homepage="https://crates.io/"
 distfiles="https://github.com/rust-lang/cargo/archive/${version}.tar.gz"
-checksum=d14462641610a56fd444200b02bb2547b663de3e63548990fba2e05435f4c096
-_cargo_dist_version=0.30.0
```

Comment:
I've removed `_cargo_dist_version` here, is there ever a case where we want an older cargo for bootstrapping?

Reply:
Ah, nevermind, we want that for when you build the initial musl tooling